Glextensions.h

```diff
@@ -13,7 +13,9 @@
 #ifndef __GLEXTENSIONS__
 #define __GLEXTENSIONS__
 
+#ifndef GL_GLEXT_VERSION
 #include "glext.h"
```

Comment:
Checking SDL_opengl.h which also triggers some redefinition warnings:
 *  It is included here because glext.h is not available on some systems.
 *  If you don't want this version included, simply define "NO_SDL_GLEXT"

I think reason is similar here, to provide the file for systems that do not have it. Nowadays, I cannot think of any relevant ones in that case. I will probably put a follow-up change to clean local glext.h and set NO_SDL_GLEXT (if I get complaint that can be easily reverted)